asyncapi/twilio/Module.md

```diff
@@ -1,31 +1,34 @@
 ## Overview
 
 The Twilio Trigger allows you to listen to Twilio SMS and Call status change events.
-1. Listen to incoming message events and message status change callback events from the twilio SMS.
-2. Listen to incoming call events and call status change callback events from the twilio Voice Call.
+1. Listen to incoming message events and message status change callback events from the Twilio SMS.
+2. Listen to incoming call events and call status change callback events from the Twilio Voice Call.
 
 This module supports [Twilio Basic API 2010-04-01](https://www.twilio.com/docs/all) version.
 
 ## Prerequisites
 Before using this trigger in your Ballerina application, complete the following:
 
-* Sign up to twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
+* Sign up to Twilio and create a Twilio account (https://support.twilio.com/hc/en-us/articles/360011177133-View-and-Create-New-Accounts-in-Twilio-Console).
 * Get a twilio phone number which needs to purchase through Twilio to send messages or make phone calls using Twilio.
     1. Navigate to the Phone Numbers page in your console.
```

Comment:
Is `Phone Numbers` page correct. [1] says it is `Buy a Number` page. So do we need to change?
Suggestion if we are to change:
"Navigate to the **Buy a Number** page in the Twilio console."

[1] https://support.twilio.com/hc/en-us/articles/223135247-How-to-Search-for-and-Buy-a-Twilio-Phone-Number-from-Console

Reply:
When login to the console there is a section called `Phone Numbers` in the left pannel